### PR TITLE
Add LIT Staking

### PR DESCRIPTION
**Pull request by SteffenKeller**
### Commitment

By submitting this PR I agree that I will fight for the full decentralization of Cardano by only running a single pool and reporting members that are breaking the rules by running multiple pools.

#### PR Checklist

I have:
- [x] Added a telegram handle of an **individual** and not that of a group or channel.
- [x] Added a social media account with activity.
- [x] Removed any `@` characters from Telegram, Instagram or Twitter handles.
- [x] Provided my operator name as my real full name, my real first name, or I have left that field blank.
- [x] Included all social media fields in the PR even if they are blank (this will fail validation if not done).

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -5419,5 +5419,19 @@
...
+   "ticker": "LIT",
+   "poolId": "02c7fc9aa1f415c8596f0957f68a3908dbdf458da4d09be93ddd608b",
+   "social": {
+     "twitter": "LitStaking",
+     "website": "https://litstaking.com",
+     "facebook": "",  
+     "instagram": "",
+     "youtube": ""
+   },
+   "telegram": "",
```
Comment (Bez625) [inline registry.json:5433]: You must provide a telegram handle and it must be that of an individual and not a group or channel

**Review by Bez625 — CHANGES_REQUESTED** [review]

File: registry.json (hunk 1 above)
Comment (SteffenKeller) [reply registry.json]: Added the Telegram handle. Thank you!

**Comment by Bez625** [comment]
The twitter account doesnt seem to have any activity or followers or following. Please can we see some social media presence outside of a blank twitter profile?

**Comment by SteffenKeller** [comment]
The pools Twitter is brand new. My personal Twitter is @0xRabbit.

**Comment by Bez625** [comment]
> The pools Twitter is brand new. My personal Twitter is 0xRabbit.

Well you addressed that pretty quickly. I'll approve now

**Review by Bez625 — APPROVED** [review]

**Pull request merged**